node_modules/photon-colors/colors.scss

```diff
@@ -0,0 +1,65 @@
+/* Firefox Colors SCSS Variables v1.0.3 */
```

Comment:
Files in node_modules should not be checked in, both `npm install` and `npm ci` would clobber your changes.

Instead, change the colors here: https://github.com/shreya99oak/fxa-content-server/blob/1acf910ec4b0ba8c54f4e7a5356b76a4debb2fb7/app/styles/_variables.scss#L131:L132